storage/spanstore/interface.go

```diff
@@ -50,6 +50,13 @@ type Reader interface {
 	FindTraceIDs(ctx context.Context, query *TraceQueryParameters) ([]model.TraceID, error)
 }
 
+// TraceGetParameters contains parameters of a trace get.
+type TraceGetParameters struct {
+	TraceID   model.TraceID
+	StartTime *time.Time
+	EndTime   *time.Time
```

Comment:
time.Time has `IsZero` that can be used as an indication of "value not provided", instead of pointers.